### PR TITLE
Fast API version

### PR DESCRIPTION
**Pull request by radekaadek**
API container using Fast API and conda

### CODE REVIEW

File: containerized_ml_models/api/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM continuumio/anaconda3
+
+COPY model.pkl requirements.txt api.py ./
+
+EXPOSE 8000
+
+RUN conda update -n base conda && \
+    conda create -n venv python=3.9 \
+    conda install scikit-learn \
```
Comment (Pawlo77) [inline containerized_ml_models/api/Dockerfile:9]: There is no need to install scikit-learn manually, you are passing it again to pip via requirements.txt

File: containerized_ml_models/api/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    conda run -n venv pip install -r requirements.txt
+
+ENTRYPOINT [ "conda", "run", "-n", "venv", "python", "-m", "api" ]
```
Comment (Pawlo77) [inline containerized_ml_models/api/Dockerfile:12]: No newline at the end of the file 😞

File: containerized_ml_models/api/api.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+@app.post("/predict")
+def predict():
+    request_data = Request.get_json()
+    data = np.array([request_data["data"]])
+    prediction = model.predict(data)
+    return {"Result": prediction.tolist()}
+
+
+if __name__ == "__main__":
+    run(app, host="0.0.0.0", port=8000)
```
Comment (Pawlo77) [inline containerized_ml_models/api/api.py:24]: No newline at the end of the file 😞

**Review by Pawlo77 — APPROVED** [review]
Great solution ✊ 

Specifying versions for all dependencies in requirements.txt might make it more future-proof, thus there might be some time when the support for your use case will be dropped (ex newest numpy won't work with that old xgboost).

File: containerized_ml_models/api/api.py (hunk 3 above)
Comment (radekaadek) [reply containerized_ml_models/api/api.py:24]: There is a newline here but github just doesnt show it, its indicated by this symbol: 
image

**Review by matined — APPROVED** [review]

**Pull request merged**